.gitignore

```diff
@@ -5,17 +5,28 @@
 /dist/
 /wheelhouse/
 /local/
-.vscode/
 .gdb_history
 dlite.egg-info
 
+# Ignore Python Files
+/.venv
+/venv
+*.pyc
+
 # Ignore personal modified bootstrap files
 /bootstrap-win-*.sh
 
 # For Visual Studio
 /.vs
 /CMakeSettings.json
 
+# For VS Code
+/.vscode
+*.code-workspace
+
+# For CTest
+/Testing/*
```

Comment:
Best practice is to run cmake from a separate build directory. In that case /Testing/ will be in that directory and not needed in .gitignore.